Assets/MRTK/Core/Providers/BaseCameraSettingsProvider.cs

```diff
@@ -49,6 +49,15 @@ public virtual void ApplyConfiguration()
                 CameraCache.Main.farClipPlane = cameraProfile.FarClipPlaneTransparentDisplay;
                 QualitySettings.SetQualityLevel(cameraProfile.TransparentQualityLevel, false);
             }
+
+            if(Application.isPlaying)
+            {
+                // Move the camera upwards by FloorHeight units if the experience settings explicitly have MRTK initialize the camera to floor height
```

Comment:
Do you see a difference between moving the camera upwards vs (in the senario where you know the camera is starting at the origin) placing a floor plane `FloorHeight` units down?

Reply:
This also might simplify the matrix of adjustments a bit:

| Room scale supported      | Room scale not supported (fall back to origin at head)|
| ----------- | ----------- |
| Adjust content up to align with camera | Content is in the right place (since the editor has the camera at the origin typically)       |
| Floor is in the right place at (0,0,0)   | Floor is adjusted down to `FloorHeight`        |

as opposed to needing to adjust the content in both cases possibly?